analysis/src/SignatureHelp.ml

```diff
@@ -0,0 +1,213 @@
+type cursorAtArg = Unlabelled of int | Labelled of string
+
+let signatureHelp ~path ~pos ~currentFile ~debug =
+  let posBeforeCursor = (fst pos, max 0 (snd pos - 1)) in
```

Comment:
This looks very specific -- copy paste from somewhere else right. Worth extracting into a function?

Reply:
Correct, copy paste from autocomplete! Definitively worth extracting to its own function.